src/Modules/ModParser.lua

```diff
@@ -10,6 +10,10 @@ local band = bit.band
 local bor = bit.bor
 local bnot = bit.bnot
 
+local function upperCaseFirstLetter(str)
```

Comment:
Consider renaming this function to `capitalise`. Most programming languages should have some function named like that in std/common 3rd-party libraries. Compare and constrast with `titleCase`, where every word is capitalised.

Reply:
Decided to just remove the function and inline it, since we were doing that elsewhere in ModParser, anyway